doc-Deployment_Planning_Guide/topics/planning.adoc

```diff
@@ -259,6 +259,44 @@ Also see https://access.redhat.com/documentation/en-us/reference_architectures/2
 For information on configuring database failover in VMDB appliances, see the https://access.redhat.com/documentation/en-us/red_hat_cloudforms/4.7/html-single/high_availability_guide/[_High Availability Guide_].
 endif::cfme[]
 
+[[database-configuration]]
+
+=== Database Configuration
+
+This section describes the {product-title} PostgreSQL database configuration. The below table provides information on each file: its location, primary function, and notes regarding behavior or recommendations. 
+
+.Database files
+[width="100%",cols="20%,20%,30%,30%",options="header",]
+|===
+|File|Location|Description|Note
+|`postgresql.conf` |Data Directory(exact location?)|Default server configuration|Adds directive to include `/etc/manageiq/postgresql.conf.d` directory
```

Comment:
This is in the `$APPLIANCE_PG_DATA` environment variable on appliances, but for this version is `/var/opt/rh/rh-postgresql95/lib/pgsql/data`

Keep in mind this *will* change in the next version of CF/ManageIQ so I'm not sure it's worth putting in here. `Data Directory` is probably good enough.

Reply:
Sounds good, thanks for the clarification and I agree, if its going to change, that reference is sufficient.